recipe/meta.yaml

```diff
@@ -72,4 +73,4 @@ extra:
     recipe-maintainers:
         - kwilcox
         - pmlandwehr
-        - sodre
+- sodre
```

Comment:
It seems like this line change is causing issues with rerender. if you revert it, that should fix it

Reply:
Aff... tks. Sorry to bother you with such dumb issue